### DISCUSSION TITLE
Wiki update

### DISCUSSION THREAD

**Discussion by ghost** [Universal-Debloater-Alliance/universal-android-debloater-next-generation#248] [General]
Link to the *An Analysis of Pre-installed Android Software* (worth reading at the bottom of the page) in wiki doesn't work, should be updated
Here are alternative links to the article
- [direct pdf](https://dspace.networks.imdea.org/bitstream/handle/20.500.12761/684/An_Analysis_of_Pre-installed_Android_Software_2019_EN.pdf?sequence=1)
- [researchgate](https://www.researchgate.net/publication/332932516_An_Analysis_of_Pre-installed_Android_Software)

**Comment by AnonymousWP** [comment]
Thanks, fixed: https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/wiki/Home/_history
